docs/molgenis/use_analytics.md

```diff
@@ -2,15 +2,15 @@
 
 ## Introduction
 
-Currenty analytics only support [google analytics](https://analytics.google.com/analytics/web/)
+Currently analytics only support [google analytics](https://analytics.google.com/analytics/web/)
```

Comment:
huh, you just showed it also support matamo? Or is this unrelated?

Reply:
unrelated, this is the framework used to also send the site improve analytics to our own backend. I was noodling around in this code to find out whether I needed to use it for this PR and saw a typo. (it was not needed)